language/types_values_variables.md

```diff
@@ -269,7 +274,7 @@ Represents the abstract notion of "value", its subtypes are `Numeric`, `String`
 
 ### Numeric
 
-Represents the abstract notion of "number", its subtypes are `Integer`, and `Float`.
+Represents the abstract notion of "number", its subtypes are `Integer`, `Float`, `Timespan` and `Timestamp`.
```

Comment:
They break the contract for Numeric. Someone accepting Numeric may get values that will cause errors as all arithmetic operations are not applicable using the Numeric contract. They also represent values you cannot have in a catalog (yet). They are therefore not Numeric in the same sense even if they can be coerced to a representation that is. If they are truly numeric, then illogical operations like adding two Timestamps must be allowed.